src/process.cpp

```
@@ -62,6 +62,21 @@ namespace proc {
+  namespace {
+    ctx_t
+    make_desktop_app() {
+      ctx_t app;
+      app.name = std::string(DESKTOP_APP_NAME);
+      app.image_path = std::string(DESKTOP_APP_IMAGE_PATH);
+      app.id = std::to_string(DESKTOP_APP_ID);
+      app.auto_detach = true;
+      app.wait_all = true;
+      app.mouse_mode = 0;
+      app.exit_timeout = std::chrono::seconds { 5 };
+      return app;
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)